src/app/other-user-profiles/other-user-profiles.component.html

```diff
@@ -7,9 +7,12 @@ <h1>Other user skill profiles</h1>
   <mat-card *ngFor="let user of permissionOwners" class="other-user-profiles-card">
     <mat-card-content>
       <div fxLayout="row">
-        <div fxFlex>
+        <div fxFlex="90%">
```

Comment:
Is it really necessary to set `flex-basis` to `90%` here? I would avoid usage of both absolute and relative units and let Angular Material / fxFlex to decide on their own how to set an element size (if possible). Let's try to leave only `fxFlex` here and see a result.

Reply:
done